File: DailyFlash Solutions/DailyFlash Solutions/01-DailyFlash_Solutions/14_Jan_Solutions_Two/java/CheckAlphaDigitSpecial.java
/*

Program 3: Write a Program to Find whether the character is an alphabet, a
digit or a special character
Input: *
Output: * is a Special Character.

*/



class CheckCharacter{
	
	public static void main(String args[]){
	
		char ch= '@';

        	if((ch>=65&&ch<=90)||(ch>=97&&ch<=122))
                	System.out.println(ch+" is alphabet");
        	else if(ch>=48&&ch<=57)
                	System.out.println(ch+" is digit");
        	else
                	System.out.println(ch+" is special char");



	}	

}	

File: DailyFlash Solutions/DailyFlash Solutions/01-DailyFlash_Solutions/14_Jan_Solutions_Two/python/checkalphadigitspecial.py
"""

Program 3: Write a Program to Find whether the character is an alphabet, a
digit or a special character
Input: *
Output: * is a Special Character.

"""


ch='5'

if (ch>='a'and ch<='z')or(ch>='A' and ch<='Z'):
    print(ch+" is alphabet")
elif ch>='0' and ch<=9:
    print(ch+" is digit")
else:
    print(ch+" is special character")
